pkg/controller/windowsmachineconfig/windows/windows.go

```diff
@@ -217,6 +220,47 @@ func (vm *Windows) waitForHybridOverlayToRun() error {
 	return fmt.Errorf("timeout waiting for hybrid-overlay: %v", err)
 }
 
+// ConfigureCNI ensures that the cni configuration in done on the node
+func (vm *Windows) ConfigureCNI(configFile string) error {
+
+	// create cni directory
+	_, _, err := vm.Run(mkdirCmd(cniDir), false)
+	if err != nil {
+		return errors.Wrapf(err, "unable to create CNI directory %v", cniDir)
+	}
+	// copy the cni plugins to windows VM
+	var cniPlugins = []string{
+		wkl.FlannelCNIPluginPath,
+		wkl.WinBridgeCNIPlugin,
+		wkl.HostLocalCNIPlugin,
+		wkl.WinOverlayCNIPlugin,
+	}
```

Comment:
Can't you add the `configFile` to list and make the error generic? Then you won't need to copy the config file separately.